lib/ansible/module_utils/facts/hardware/linux.py

```diff
@@ -21,28 +21,22 @@
 import json
 import os
 import re
-import signal
 import sys
 import time
 
-from multiprocessing import cpu_count
-from multiprocessing.pool import ThreadPool
-
-from ansible.module_utils.common.text.converters import to_text
+from ansible.module_utils._internal._concurrent import _futures
 from ansible.module_utils.common.locale import get_best_parsable_locale
+from ansible.module_utils.common.text.converters import to_text
```

Comment:
The original import above can be kept instead of moving it here.

Reply:
its easier to find/update stuff if they are ordered